PROVESFlightControllerReference/test/int/0_radio_test.py

```
@@ -0,0 +1,57 @@
+from common import proves_send_and_assert_command
+from fprime_gds.common.models.serialize.time_type import TimeType
+from fprime_gds.common.testing_fw.api import IntegrationTestAPI
+
+downlinkDelay = "ReferenceDeployment.downlinkDelay"
+lora = "ReferenceDeployment.lora"
+
+LORA_ERROR_EVENTS = ("SendFailed", "ConfigurationFailed", "AllocationFailed")
+
+
+@pytest.fixture(autouse=True)
+def setup_test(fprime_test_api: IntegrationTestAPI, start_gds):
```

Ruff (0.15.12)

[warning] Unused function argument: `start_gds`

(ARG001)